examples/hello-world-service/hello_world_service.md

```diff
@@ -1,10 +1,13 @@
 # Hello world service
 
-Let's write a simple HTTP service in Ballerina. 
-This example demonstrates the network primitives in the language that make it simpler to develop services.
+Let's write a simple HTTP service in Ballerina. This example demonstrates the network primitives in the language that make it simpler to develop services.
```

Comment:
Didn't you mention a 100 column limit for line length?

Reply:
That is only for the BAL file.